### PR TITLE
Add image cropping functionality using Cropper.js

### PR DESCRIPTION
**Pull request by lgaleana-llm**
Added the ability to crop scraped images in the UI using the Cropper.js library. This includes modifications to the HTML, JavaScript, and CSS files to integrate the cropping functionality. Additionally, unit and E2E tests were added to ensure the cropping functionality works as expected.

### Test Plan

pytest / playwright

### CODE REVIEW

**Comment by lgaleana-llm** [comment]
!!!!! WARNING: Unrelated tests aren't passing !!!!!

**Comment by lgaleana-llm** [comment]
Total time to create PR: 218.93 seconds
Total OPenAI cost: 0.6559699999999999

File: templates/static/scripts.js [hunk 1]
```diff
@@ -27,6 +31,18 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
                 headline.setAttribute('data-y', 10); // Set initial y position
                 container.appendChild(headline);
 
+                const cropButton = document.createElement('button');
```
Comment (lgaleana) [inline templates/static/scripts.js:34]: How does this button work? I want to be able to drag the edges of the image to crop it.

Comment (lgaleana-llm) [reply templates/static/scripts.js:34]: The crop button initializes the Cropper.js library, which allows you to drag the edges of the image to crop it. When you click the "Crop" button, the image is displayed in a cropping interface where you can adjust the crop area by dragging the edges. Once you're satisfied with the crop, you can click the "Crop" button again to apply the crop and update the image.

**Comment by lgaleana-llm** [comment]
Total time to amend PR: 16.11 seconds
Total OPenAI cost: 0.092435

**Comment by lgaleana-llm** [comment]
The issue seems to be related to the download event timing out. I'll investigate the root cause and ensure that the download event is properly triggered and handled within the test. I'll update the test to ensure it waits correctly for the download event to complete.

**Comment by lgaleana-llm** [comment]
Total time to amend PR: 456.57 seconds
Total OPenAI cost: 1.813845

**Review by lgaleana — APPROVED** [review]

**Pull request merged**